Bottom: a088c7bc18db94b67dd868a40d54ef2dbefeb446
Top:    0f088d7adf4612f10522695d406631a18b9bd53f
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-08-02 16:03:19 +0800

Refresh of tap-support-batch-receiving

---

diff --git a/drivers/net/tap.c b/drivers/net/tap.c
index aaef87b..17f0d33 100644
--- a/drivers/net/tap.c
+++ b/drivers/net/tap.c
@@ -1221,7 +1221,7 @@ static int tap_sendmsg(struct socket *sock, struct msghdr *m,
 			xdp = &((struct xdp_buff *)ctl->ptr)[i];
 			tap_get_user_xdp(q, xdp);
 		}
-		return total_len;
+		return 0;
 	}
 
 	return tap_get_user(q, m, &m->msg_iter, m->msg_flags & MSG_DONTWAIT);
